contents/graham_scan/code/c++/graham_scan.cpp

```diff
@@ -0,0 +1,66 @@
+#include<iostream>
+#include<vector>
+#include<algorithm>
+
+struct point{
+  float x;
+  float y;
+};
+
+bool ccw(point a, point b, point c){
+  return ((b.x - a.x)*(c.y - a.y) >= (b.y - a.y)*(c.x - a.x));
+}
+
+std::vector<point> grahamScan(std::vector<point> points){
+  //selecting lowest point as pivot
+  int lowIndex = 0;
+  for (size_t i = 1; i < points.size(); i++) {
+    if(points[i].y < points[lowIndex].y){
+      lowIndex = i;
+    }
+  }
+  std::swap<point>(points[0], points[lowIndex]);
+  point pivot = points[0];
+
+  //sorting points by polar angle
+  sort(points.begin()+1, points.end(), [pivot](point a, point b){
```

Comment:
`sort` is in namespace `std`.
```suggestion
  std::sort(points.begin()+1, points.end(), [pivot](point a, point b){
```